src/client/Client.js

```diff
@@ -135,7 +135,8 @@ class Client extends BaseClient {
     Object.defineProperty(this, 'token', { writable: true });
     if (!browser && !this.token && 'DISCORD_TOKEN' in process.env) {
       /**
-       * Authorization token for the logged in bot
+       * Authorization token for the logged in bot.
+       * Gets set to `process.env.DISCORD_TOKEN` when instantiating the client
```

Comment:
Maybe something like this would sound more..clear (so users understand it is set to that only if it is present in the env). Wording can be improved 😅 

```suggestion
       * Gets set to `process.env.DISCORD_TOKEN` when instantiating the client and it is present
```

Reply:
Alternative wording:
```suggestion
       * Defaults to `process.env.DISCORD_TOKEN` when instantiating the client if it was defined
```